webapp/src/shared/context/auth-context.tsx

```diff
@@ -28,6 +28,16 @@ export const AuthProvider = ({ children }: { children: ReactNode }) => {
   }, []);
   const { data, isLoading: isDataLoading } = useData<User>(getCurrentUser);
   const navigate = useNavigate();
+  const authBroadcastChannel = useMemo(
+    () => new BroadcastChannel('auth_channel'),
+    [],
+  );
```

Comment:
Is this available to any webapp instance running on the same browser?

Reply:
Yes, each application running in a different tab or window subscribes to the `auth_channel` -> https://developer.mozilla.org/en-US/docs/Web/API/BroadcastChannel#constructor

And they can send messages to each other.